azdev/operations/command_change/__init__.py

```diff
@@ -95,6 +95,8 @@ def export_command_meta(modules=None, git_source=None, git_target=None, git_repo
             "is_preview": command.command_kwargs.get("is_preview", False)
         }
         module_loader = command_loader.cmd_to_loader_map[command_name]
+        for loader in module_loader:
+            loader.skip_applicability = True
```

Comment:
Can we add some description on how this fixes the https://github.com/Azure/azure-cli/pull/26572#discussion_r1211377824?